[groheondus] account connection issue

Since some time I got no connection to ONDUS account, so all related things are broken.

In Paper UI I see for the GROHE ONDUS Account:
Status: UNINITIALIZED - HANDLER_INITIALIZING_ERROR org/apache/http/impl/client/HttpClients

I see no further info in the error log.

Used version: SNAPSHOT 2.5.0 (#1619), last updated 21.06.2019

I'm using **openHAB 2.5.0~S1645-1 (Build #1645)**. Updating regularly by apt-get.
Recently I stopped openHAB, cleaned cache and restarted openHAB.

In the log I now see:
_2019-07-26 16:44:22.506 [hingStatusInfoChangedEvent] - 'groheondus:account:6a1a10b5' changed from UNINITIALIZED to INITIALIZING

2019-07-26 16:44:22.601 [hingStatusInfoChangedEvent] - 'groheondus:account:6a1a10b5' changed from INITIALIZING to UNKNOWN

==> /var/log/openhab2/openhab.log <==

2019-07-26 16:44:22.821 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.groheondus.internal.handler.GroheOndusAccountHandler@1ed8cb5': org/apache/http/impl/client/HttpClients

java.lang.NoClassDefFoundError: org/apache/http/impl/client/HttpClients

	at org.grohe.ondus.api.client.ApiClient.<init>(ApiClient.java:30) ~[?:?]

	at org.grohe.ondus.api.OndusService.login(OndusService.java:34) ~[?:?]

	at org.openhab.binding.groheondus.internal.handler.GroheOndusAccountHandler.initialize(GroheOndusAccountHandler.java:73) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]

	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [140:org.openhab.core:2.5.0.201907230301]

	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [140:org.openhab.core:2.5.0.201907230301]

	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]

	at java.lang.Thread.run(Thread.java:748) [?:?]

Caused by: java.lang.ClassNotFoundException: org.apache.http.impl.client.HttpClients cannot be found by ondus-api_0.0.9

	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:433) ~[?:?]

	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[?:?]

	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[?:?]

	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[?:?]

	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:?]

	... 13 more

==> /var/log/openhab2/events.log <==

2019-07-26 16:44:22.935 [hingStatusInfoChangedEvent] - 'groheondus:account:6a1a10b5' changed from UNKNOWN to UNINITIALIZED (HANDLER_INITIALIZING_ERROR): org/apache/http/impl/client/HttpClients

==> /var/log/openhab2/openhab.log <==

2019-07-26 16:44:22.925 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'groheondus:account:6a1a10b5': org/apache/http/impl/client/HttpClients

java.lang.NoClassDefFoundError: org/apache/http/impl/client/HttpClients

	at org.grohe.ondus.api.client.ApiClient.<init>(ApiClient.java:30) ~[?:?]

	at org.grohe.ondus.api.OndusService.login(OndusService.java:34) ~[?:?]

	at org.openhab.binding.groheondus.internal.handler.GroheOndusAccountHandler.initialize(GroheOndusAccountHandler.java:73) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]

	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [140:org.openhab.core:2.5.0.201907230301]

	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [140:org.openhab.core:2.5.0.201907230301]

	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]

	at java.lang.Thread.run(Thread.java:748) [?:?]

Caused by: java.lang.ClassNotFoundException: org.apache.http.impl.client.HttpClients cannot be found by ondus-api_0.0.9

	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:433) ~[?:?]

	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[?:?]

	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[?:?]

	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[?:?]

	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:?]

	... 13 more

==> /var/log/openhab2/events.log <==

2019-07-26 16:44:22.994 [hingStatusInfoChangedEvent] - 'groheondus:senseguard:1938d9da-af4f-4e81-a605-37c37e5d5728' changed from UNINITIALIZED to INITIALIZING_

--- a/bundles/org.openhab.binding.groheondus/pom.xml
+++ b/bundles/org.openhab.binding.groheondus/pom.xml
@@ -17,7 +17,7 @@
     <dependency>
       <groupId>org.grohe</groupId>
       <artifactId>ondus-api</artifactId>
-      <version>0.0.9</version>
+      <version>0.0.10</version>
     </dependency>
   </dependencies>
 

--- a/bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml
+++ b/bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml
@@ -7,9 +7,9 @@
         <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-core/2.9.6</bundle>
         <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/2.9.6</bundle>
         <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/2.9.6</bundle>
-        <bundle dependency="true">mvn:org.apache.httpcomponents/httpcore-osgi/4.4.9</bundle>
-        <bundle dependency="true">mvn:org.apache.httpcomponents/httpclient-osgi/4.5.5</bundle>
-        <bundle dependency="true">mvn:org.grohe/ondus-api/0.0.9</bundle>
+        <bundle dependency="true">mvn:org.apache.commons/commons-text/1.6</bundle>
+        <bundle dependency="true">mvn:org.apache.commons/commons-lang3/3.8.1</bundle>
+        <bundle dependency="true">mvn:org.grohe/ondus-api/0.0.10</bundle>
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.groheondus/${project.version}</bundle>
     </feature>
 </features>

